dev-libs/serdisplib/files/disable-static-build.patch

```diff
@@ -0,0 +1,212 @@
+--- a/src/Makefile.in	2017-02-02 13:37:48.000000000 +0100
++++ b/src/Makefile.in	2018-05-14 20:52:34.061230980 +0200
+@@ -65,7 +65,6 @@
+ 
+ LIB_NAME=serdisp
+ 
+-LIB_STATIC=lib$(LIB_NAME).a
```

Comment:
I don't really understand why you're removing all the static logic here. As long as it's turned off, keeping it doesn't hurt and there's no reason to use a large patch to remove something that is not being used.

Reply:
The static lib is always build, as the default tools and USE="tools" extra tools depend on the static build always. So I changed it, to remove the whole static stuff, as I don't see anyway to do that? Do you disagree?